### PR TITLE
Switch to `child-src` and move requires along with the rest.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,21 +6,24 @@
...
 var handlebars = require('handlebars');
 handlebars = require('handlebars-helper-sri').register(handlebars);
 
 var helpers = require('./lib/helpers.js');
 
-var CSP_HEADER = "default-src 'none'; img-src 'self'; style-src 'self'; font-src 'self' ; frame-src 'self'"; // jshint ignore:line
-
 var server = new Hapi.Server();
+
+var CSP_HEADER = 'default-src "none"; img-src "self"; style-src "self"; font-src "self"; child-src "self"';
```
Comment (fmarier) [inline index.js]: We probably need to keep `frame-src` in there too for browsers that don't support `child-src`.

Comment (XhmikosR) [inline index.js]: But, do you really care about those browsers?

Comment (mozfreddyb) [inline index.js]: We care about webcompat and users. That's kind of our thing :stuck_out_tongue: 

Which browsers are that and are they among the auto-update browsers?
I'd suppose Safari/Webkit on OS X and iOS devices.

Comment (XhmikosR) [inline index.js]: Don't get me wrong, I'm all for compatibility.

But this is a security measure. If one doesn't keep up with the latest browsers, it's fair to say they don't want to be as secure as possible.

That's my personal opinion of course :)

Comment (fmarier) [inline index.js]: Safari and Edge don't support CSP 2 but support CSP 1 so we should leave `frame-src` in there for backwards compatibility.

Also `frame-src` has been [un-deprecated in CSP3](https://w3c.github.io/webappsec-csp/#directive-frame-src).

Comment (XhmikosR) [inline index.js]: If it has been un-deprecated, I'd say this change is moot then.

**Pull request closed without merging**